content/blogs/_index.md

```diff
@@ -7,4 +7,193 @@ aliases: [/blog]
 logo: /img/logo-sm.svg
 ---
 
-{{% children %}}
+##### August 9, 2018
+
+#### [OpenCensus for IoT](/articles/iot)
+
+Our application is a prototype which demonstrates how we can utilize OpenCensus for the observability in the IoT industry. With the service of metric collection and monitoring provided by OpenCensus, we’ll gain the insights into hardware and software performance across distributed IoT devices... [\[read more\]](/blogs/iot)
+
+---
+
+##### July 11, 2018
+
+#### [Monitoring HTTP Latency with OpenCensus and Stackdriver](https://medium.com/google-cloud/monitoring-http-latency-with-opencensus-and-stackdriver-bf561608e81a)
+
+This post will describe how to code your own monitoring probes, similar in function to Stackdriver uptime checks on Google Cloud. Code, configuration files, commands, and detailed instructions… [\[read more\]](https://medium.com/google-cloud/monitoring-http-latency-with-opencensus-and-stackdriver-bf561608e81a)
+
+---
+
+##### June 13, 2018
```

Comment:
@hvent90 in deed, as @songy23 mentioned, please remove the articles that I posted up as those are under guides :)